src/styles/main.scss

```diff
@@ -1,3 +1,45 @@
-body {
-  margin: 0;
+@import "./reset.scss";
+@import "./variables.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $calendar-pading;
+  width: $day-width * 7 + $calendar-pading * 2 + $gap * 6;
+
+  &__day {
+    display: flex;
+    background-color: #eee;
+    height: $day-height;
+    width: $day-width;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.5s;
+    border: 1px solid #000;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
the cursor must be always when user hovers day block, so move it to the main selector